composer-distributor.xsd

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema attributeFormDefault="unqualified" elementFormDefault="qualified"
+           targetNamespace="https://phar.io/composer-distributor" xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:rep="https://phar.io/composer-distributor">
+
+  <xs:element name="composer-distributor" type="rep:distributorType"/>
```

Comment:
Even though it's of course perfectly valid XML, I personally have a dislike for `-` within a tag name. Maybe we can (should?) shorten the root element name to `distributor`? That would also match the filename...

Reply:
I've just seen that the w3c uses camelCased names (Or how are the camelCases called that don't have a first capital letter?). So perhaps `composerDistributor` would be an option?

Reply:
Changed everything to `distributor` only :)